docs/icons/TwitterIcon.tsx

```diff
@@ -0,0 +1,8 @@
+export const TwitterIcon = () => (
```

Comment:
Hmm, why are we doing this rather than using `svg`s?

Reply:
Sure! I’ve set this PR into the state where I’m getting the error. In `<Footer />` I’m importing `<TwitterIcon />` and trying to use it, but I get: `Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object`.

https://github.com/glow-xyz/nftoken/blob/a10ce7bac4b18747f9c02e9ce0cd41d86d7e56a8/docs/components/Footer.tsx#L1-L20

Reply:
@benborgers pushed a fix in 08762d2a40d97fd9df1ae79335a753912245dbf7

Reply:
Thank you!